src/popup/popup.tsx

```diff
@@ -455,13 +429,7 @@ export default class Popup<P extends BasePopupProps = PopupProps> extends PureCo
                     [styles.largeBorderRadius]: largeBorderRadius,
                   });
                   return (
-                    <span
-                      // prevent bubbling through portal
-                      onClick={stop}
```

Comment:
If I understand correctly, after this change, some clicks may unexpectedly start bubbling from popups. I manually went through the YouTrack interface, but couldn't find any problems. But we need to be prepared for this.